feed.json.liquid

```diff
@@ -0,0 +1,55 @@
+---
+permalink: feed.json
```

Comment:
👏🏻 this file is great. Almost wonder about it how hard it would be to make it a Jekyll plugin one day?